src/main/java/com/bitmovin/api/encoding/codecConfigurations/AV1VideoConfiguration.java

```diff
@@ -0,0 +1,159 @@
+package com.bitmovin.api.encoding.codecConfigurations;
+
+
+public class AV1VideoConfiguration extends VideoConfiguration {
+
+    private Integer lagInFrames;
+    private Integer minQ;
+    private Integer maxQ;
+    private Integer undershootPct;
+    private Integer overshootPct;
+    private Long clientBufferSize;
+    private Long clientOptimalBufferSize;
+    private Integer tileColumns;
+    private Integer tileRows;
+    private Boolean isAutomaticAltRefFramesEnabled;
+    private Integer arnrMaxFrames;
+    private Integer arnrStrength;
+    private Long maxIntraRate;
+    private Boolean isLossless;
+    private Boolean isFrameParallel;
+    private Integer sharpness;
+    private Boolean isFrameboostEnabled;
+    private Boolean noiseSensitivity;
+    private Integer minGfInterval;
+    private Integer maxGfInterval;
+    private Integer numTileGroups;
+    private Integer mtuSize;
+
+
+    public Integer getLagInFrames() { return this.lagInFrames; }
```

Comment:
Please write getters and setters not as one-liner.
`{` always on a new line. After that the statement, and then the `}` on a new line again.